libsql/src/replication/connection.rs

```diff
@@ -566,10 +570,7 @@ impl Stmt for RemoteStatement {
     }
 }
 
-pub(crate) struct RemoteRows(
-    pub(crate) ResultRows,
-    pub(crate) usize,
-);
+pub(crate) struct RemoteRows(pub(crate) ResultRows, pub(crate) usize);
```

Comment:
is this rustfmt making this change? Seems odd since I should be running it on every save.

Reply:
Apparently so. I have fmt running on every file save.